samples/astarte_app/include/utils.h

```diff
@@ -45,7 +45,7 @@ extern const char *const utils_string_array_data[2];
  *
  * @param[in] individual The individual to log
  */
-void utils_log_astarte_individual(astarte_individual_t individual);
+void utils_log_astarte_individual(astarte_data_t individual);
```

Comment:
Change also in the doc
```suggestion
void utils_log_astarte_individual(astarte_data_t data);
```